response.go

```diff
@@ -1,85 +1,98 @@
 package slacker
 
 import (
-	"fmt"
-	"log"
-
 	"github.com/slack-go/slack"
 )
 
-const (
-	errorFormat = "*Error:* _%s_"
-)
+// WriterReplierResponse interface is used to respond to an event
+type WriterReplierResponse interface {
```

Comment:
Needs docs, and does the interface give us any benefit here? If it's needed, why not just embed the interfaces into a super interface like:

```
type WriterReplierResponse interface {
  Replier
  WriterResponse
}
```

Reply:
Cleaned it up